runtime/doc/lsp.txt

```diff
@@ -25,24 +25,37 @@ Nvim provides an LSP client, but the servers are provided by third parties.
 Follow these steps to get LSP features:
 
 1. Install language servers using your package manager or by following the
-   upstream installation instruction. You can find language servers here:
+   upstream installation instructions. You can find language servers here:
    https://microsoft.github.io/language-server-protocol/implementors/servers/
```

Comment:
Would it make sense to link to `nvim-lspconfig` instead?

Reply:
Not all users will have nvim-lspconfig installed, and in any case the hope here is that we can get more users to use LSP without it.

Reply:
I was not suggesting replacing this by "use `nvim-lspconfig`", but to use their list of available servers instead since I think it will better reflect which LSPs currently have neovim support.
I don't have strong feelings about it though so it's fine.